CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing to Azure DevOps Extension
+
+Welcome, and thank you for your interest in contributing to Azure DevOps Extension!
+
+There are many ways in which you can contribute, beyond writing code. The goal of this document is to provide a high-level overview of how you can get involved.
+
+# Have a question
+
+Search existing github issues for similar questions and feel free to file an issue with a "question" tag to get our attention. 
+
+# Found a Bug
+
+#### Where to Find Known Issues
+
+Before you file a new bug report do check on existing open [issues](https://github.com/Microsoft/vsts-cli/issues?q=is%3Aopen+is%3Aissue+label%3Abug).
+
+If you find your issue already exists, make relevant comments and add your reaction. Use a reaction in place of a "+1" comment.
+👍 - upvote 👎 - downvote
+
+#### Reporting New Issue
+
+The best way to get your bug fixed is to provide good repro steps. Use our [Bug template](https://github.com/Microsoft/vsts-cli/issues/new/choose) to report any issues.
+
+#### Security Bugs
+
+TO-DO
+
+# Feature Requests
+
+#### Where to Find Existing Requests
+
+Before you file a new request do check on existing open [feature requests](https://github.com/Microsoft/vsts-cli/issues?q=is%3Aissue+is%3Aopen+label%3AFeature).
+
+If you find your request already exists, make relevant comments and add your reaction. Use a reaction in place of a "+1" comment.
+👍 - upvote 👎 - downvote
+
+
+#### Submit New Feature request
+
+Use our Feature Request [template](https://github.com/Microsoft/vsts-cli/issues/new/choose) to create a new feature request.
+
+# Contribute Code/Documentation Pull Requests
+
+## Your First Contribution
+
+Please go through our [code of conduct](./CODE%20OF%20CONDUCT.md) and CLA requirements.
+
+You will need to complete a Contributor License Agreement (CLA). Briefly, this agreement testifies that you are granting us permission to use the submitted change according to the terms of the project's license, and that the work being submitted is under appropriate copyright.
```

Comment:
This CLA section seems to be too jarring and potentially too scary for a new user. I am thinking how we can make this potentially more subtle.